src/CrossSigningManager.js

```diff
@@ -159,6 +162,20 @@ export const crossSigningCallbacks = {
     onSecretRequested,
 };
 
+export async function promptForBackupPassphrase() {
+    let key;
+
+    const RestoreKeyBackupDialog = sdk.getComponent('dialogs.keybackup.RestoreKeyBackupDialog');
+    const { finished } = Modal.createTrackedDialog('Restore Backup', '', RestoreKeyBackupDialog, {
+            showSummary: false, keyCallback: k => key = k,
+    }, null, /* priority = */ false, /* static = */ false);
```

Comment:
Will the 4S passphrase request dialog be triggered duing this flow? If so, set `static = true` so that restore dialog preserves state after the 4S dialog ends.

Reply:
done